profiles/mercury.raml

```diff
@@ -159,4 +160,14 @@ validations:
         pattern: ^v[0-9]+$
         minCount: 1
 
+  base-uri-matches-pattern:
+    message: The baseUri must match the pattern - https://{shortCode}.api.commercecloud.salesforce.com/<api-family>/<api-name>/{version}
+    targetClass: apiContract.WebAPI
+    functionConstraint:
+      code: |
+        function(resource) {
+          const baseUriRegex = /^https:\/\/{shortCode}.api.commercecloud.salesforce.com\/[a-z]+-?[a-z]+\/[a-z]+-?[a-z]+\/{version}$/;
```

Comment:
This does not allow a trailing / in the regex.  Is this intended?

Reply:
Yes, it is intended. If we allow a trailing / we will end up with // when the baseUri is appended with endpoints.

Reply:
Followup...does anyone currently have a trailing / in their base uri.  If not this is fine.  If so I don't think we should force people to change it right now this close to GA.

Reply:
Yeah, shopper-stores and shopper-search. I will remove the check.